benefits/settings.py

```diff
@@ -338,6 +338,7 @@ def RUNTIME_ENVIRONMENT():
     "'unsafe-inline'",
     "https://california.azureedge.net/",
     "https://fonts.googleapis.com/css",
+    "https://cdn.jsdelivr.net",
```

Comment:
I'm wondering if it is worth it to just download the Bootstrap source file and stick it in our repo?

Reply:
Yep, that makes sense. I just tested it @angela-tran but I don't think it worked because the styles did not get applied:
![image](https://github.com/user-attachments/assets/3eb70879-9e05-4859-960e-e049cff018ff)

Reply:
Putting `https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/` in `CSP_STYLE_SRC` worked! Just for documentation, if I leave out the trailing `/` it won't work.